sabnzbd/cfg.py

```diff
@@ -415,6 +415,7 @@ def validate_notempty(root, value, default):
 downloader_sleep_time = OptionNumber("misc", "downloader_sleep_time", 10, 0)
 ssdp_broadcast_interval = OptionNumber("misc", "ssdp_broadcast_interval", 15, 1, 600)
 socks5_proxy_url = OptionStr("misc", "socks5_proxy_url")
+user_defined_well_known_extensions = OptionList("misc", "user_defined_well_known_extensions", ["myext", "specialext"])
```

Comment:
There must be a shorter version 😂
To keep it similar to the other option above, what about: `ext_rename_skip`?
And why these default values? Just leave it empty and update the Wiki to have examples.